spec/01-lexical-syntax.md

```diff
@@ -59,7 +59,7 @@ idrest   ::=  {letter | digit} [‘_’ op]
 ```
 
 There are three ways to form an identifier. First, an identifier can
-start with a letter which can be followed by an arbitrary sequence of
+start with a letter that can be followed by an arbitrary sequence of
```

Comment:
Mmm, but `can be followed...` isn't restrictive of `letter`, so `which` is correct, at least that's what they told me in school.